src/graphorm/index.ts

```diff
@@ -360,16 +380,17 @@ const obj = new GraphORM({
       },
     },
   },
+  Bookmark: {
+    from: 'Bookmark',
+    fields: {
+      remindAt: { transform: transformDate },
```

Comment:
After further investigation, I saw a lot of fields here using a similar function doing `() => new Date(value)`. Before adding this here, the `remindAt` was always `null` while the `createdAt` was always correct.

Reply:
Good point. It was working when I tested it, but probably best if we include it to use the same transform function.